test/build/resolve-js/src/index.js

```diff
@@ -0,0 +1,2 @@
+import isArray from 'is-array';
```

Comment:
Also test to make sure npm packages > local package in JS (opposite of CSS). Also working as expected.